sdk/communication/test-resources.json

```diff
@@ -46,9 +72,29 @@
         }
     ],
     "outputs": {
+        "AZURE_SUBSCRIPTION_ID": {
```

Comment:
Do we need this? I think .net work without it.

Reply:
Well, Josh removed it last time from JS and it started failing. Let's sync up on your .net fix